lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -59,7 +59,9 @@
   private LottieComposition composition;
   private final LottieValueAnimator animator = new LottieValueAnimator();
   private float scale = 1f;
+  private boolean animationsEnabled = true;
```

Comment:
Let's make `animationsEnabled()` a private function and in the javadoc for `systemAnimationsEnabled` and `ignoreSystemAnimationsDisabled`, let's say `Call animationsEnabled() instead of using these fields directly`

Then we don't manually have to update a third field which could theoretically get out of sync with the other two.

Reply:
@gpeal good idea, updated PR with function and comment on fields.